js/pokemon.js

```diff
@@ -510,9 +531,11 @@ function checkPokemonLoaded() {
     if(!loadedImage.complete || loadedImage.naturalWidth == 0 || loadedImage.naturalHeight == 0) {
     
         if(++consecutiveLoadFails < 3) {
-            document.getElementById('nextCountdown').innerHTML = 'This is taking a while to load. Do you want to try loading another one? It won\'t affect your streak. <a href="#" onclick="newPokemon();">Load a new Pok&eacute;mon?</a>';
+            jQuery.data($('#nextCountdown')[0]).lang = "loadfail";
```

Comment:
I don't think this one is mine :) It was added in this PR and it doesn't look to have been moved from anywhere else in the file